modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,148 @@
+
+//         <http-policy:execute-next/>
+
+//     </http-policy:operation>
+// </http-policy:proxy>
+// ----
+
+// A policy can avoid the execution of the flow by not defining the `execute-next` or making sure that it is not reached (placing it in a choice, for example).
+
+// Variables set on a policy have only the scope of the policy. Also, variables defined in the flow, are not visible in policies.
+
+== Policy Size
+
+The size of a policy varies based on its dependencies. The first time Mule Runtime polls for policies it will take longer to fetch them. However, after policies are retrieved, they are cached on the file system. This ensures offline availability and also reduces latency to fetch policies.
+//Add more information about the policy size.
+
+
+// == RAML and OAS-Based APIs
+
+// If your API exposes a RAML or OAS definition that users might need to reference, the API definition must reliably detail every element that is expected in calls to your API. +
+// For example, applying a client ID-based policies implies that all requests coming to your API include a client ID and client Secret (by default expected as query parameters). You can prevent user requests from being rejected by creating a trait in the RAML root, and then referencing this trait in all API operations. 
+
+// Example: Create a trait that request APIs to provide Client ID and Client Secret parameters
+// [source,yaml,linenums]
+// ----
+// traits:
+//   - client-id-required:
+//       queryParameters:
+//         client_id:
+//           type: string
+//         client_secret:
+//           type: string
+// ----
+
+// Example: Apply the trait in API operations:
+
+// [source,yaml,linenums]
+// ----
+// /products:
+//   get:
+//     is: [client-id-required]
+//     description: Gets a list of all the inventory products.
+// ----
+
+== Policy Level
+
+Policies implement an additional level of granularity, where access is controlled based on the a criteria, called resource-level policies. 
+In this level of granularity, policies are applied to only those request that match the criteria.  All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
+
+//* SLA-based policies
+//Impose a limit specified by the level of access granted to the requesting application.The Rate Limiting policy is SLA-based.
+// Is SLA based a policy level granularity or a configuration level? because it doesn't change where the policy is applied, just the configuration ( a.k.a limit) - Eva??
+
+
+//[PA: We need to tell the customers how to work with SLA-tiers for policies right?]
+//SLA tiers for me should go somewhere else and link to all the documentation on contracts, request api access and those flows. - Eva?
+
+
+
+Resource level policy support includes RAML and HTTP-based APIs (since Mule 4). A policy applied at a resource level affects all HTTP methods, for example, PUT, GET, POST, or selected HTTP methods, within the resource.
```

Comment:
do we want to copy the same info that we have in the resource level policies section? probably a description and then a link on see also section?

Reply:
Done! Thanks!